ansys/dpf/core/operators/averaging/elemental_difference.py

```diff
@@ -128,9 +144,9 @@ def __init__(self, op: Operator):
 
     @property
     def field(self):
-        """Allows to connect field input to the operator
+        """Field inputs to connect to the operator.
 
-        - pindoc: field or fields container with only one field is expected
+        - pindoc: A field or fields container with only one field is expected.
```

Comment:
I'm not sure what the purpose of "pindoc :" is here. Would you please clarify what these occurrences are conveying?